rustls/src/crypto/cipher.rs

```diff
@@ -38,6 +40,42 @@ pub trait Tls12AeadAlgorithm: Send + Sync + 'static {
 pub trait MessageDecrypter: Send + Sync {
     /// Perform the decryption over the concerned TLS message.
     fn decrypt(&self, m: OpaqueMessage, seq: u64) -> Result<PlainMessage, Error>;
+
+    /// For TLS1.3 (only), checks the length m.payload is valid and removes the padding.
```

Comment:
typo, change to refer to `msg` instead of `m`.
```
    /// For TLS1.3 (only), checks the length of `msg.payload` is valid and removes the padding.
 ```
 
 alternatively, maybe `msg` should be `m` to match `decrypt`?